games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=JRomManager
+MY_PV=1.6b20
+MY_P="${MY_PN}-${MY_PV}"
+
+inherit desktop
+
+DESCRIPTION="A Mame and Retrogaming Rom Manager, Java alternative to ClrMamePro"
+HOMEPAGE="https://github.com/optyfr/JRomManager"
+SRC_URI="https://github.com/optyfr/${MY_PN}/releases/download/${MY_PV}/${MY_P}.zip -> ${P}.zip"
+QA_PREBUILT="*"
+
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=virtual/jre-1.8"
+DEPEND="${DEPEND}
+	app-arch/unzip"
+
+src_prepare() {
+	default
+
+	sed -e "/dir=.*/s:.*:cd ${EROOT}/usr/share/${PN}:" -i *.sh || die "sed failed!"
+}
+
+src_install() {
+	insinto "/usr/share/${PN}"
+	doins *.jar
+	doins -r lib
+	exeinto "/usr/share/${PN}"
+	doexe *.sh
+	dosym "${EROOT}/usr/share/${PN}/JRomManager-multi.sh" "/usr/bin/${PN}"
+	doicon "${FILESDIR}/${PN}.png"
```

Comment:
Binary files are forbidden inside the Gentoo repository. You need to fetch it via SRC_URI. Also, what's the source of that icon and its license?

Reply:
Since it's in src, I will fetch it from my github repos...
Came initially from here: http://www.urltarget.com/computer-memory-microchip-icon-electronic-rom.html which is "Public Domain CC0"

Reply:
Is there any reason it's not included in the release package?

Reply:
You can use `unzip` to extract it, can't you? ;-)

Reply:
Yes of course I can, in src_prepare() ?